src/App.css

```diff
@@ -39,74 +40,105 @@ canvas {
 .containerBody {
   /* border: 1px dotted purple; */
   display: flex;
+  /* flex-wrap: wrap; */
   flex-direction: row;
-  padding-left: 5%;
+  padding-left: 12%;
   /* overflow: hidden; */
   max-height: 70vh;
+  justify-content: space-evenly;
+  padding-top: 3%;
 }
 .canvasContainer {
   /* border: 1px solid salmon; */
   display: flex;
   flex-direction: column;
-  height: 70vh;
+  height: 100%;
+  padding: 5%;
   animation: fadeIn 3.2s ease-in;
+  /* justify-self: center; */
 }
 
 
 .swatchList {
   /* border: 1px dashed blue; */
-  height: inherit;
+  height: 97%;
```

Comment:
generally speaking, you don't want to do everything by percentage.  This tends to be really weird on extra-large or extra-small screens.